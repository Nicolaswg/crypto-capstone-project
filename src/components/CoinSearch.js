import React, { useState } from 'react';

const CoinSearch = () => {
  const [search, setSearch] = useState('');

  const onSearchChange = (e) => {
    setSearch(e.target.value);
  };

  return (
    <div className="header flex">
      <div className="flex-a flex">
        <h1>Crypto Coins</h1>
      </div>
      <div className="flex-a flex">
        <input
          type="text"
          placeholder="type your crypto"
          value={search}
          onChange={onSearchChange}
          onBlur={onSearchChange}
        />
      </div>
    </div>
  );
};

export default CoinSearch;
