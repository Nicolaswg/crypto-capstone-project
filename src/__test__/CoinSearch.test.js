import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../Redux/ConfigStore';
import CoinSearch from '../components/CoinSearch';

it('testing homepage rendering components', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Router>
        <CoinSearch />
      </Router>
    </Provider>,
  ).toJSON;
  expect(tree).toMatchSnapshot();
});
