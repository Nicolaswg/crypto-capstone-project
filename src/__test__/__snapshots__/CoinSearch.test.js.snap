// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing homepage rendering components 1`] = `[Function]`;
